src/main/java/duke/Duke.java

```diff
@@ -19,20 +19,30 @@ public static void main(String[] args) {
         System.out.println(separateLine);
 
         String line="";
-        List<String> list = new ArrayList<String>();
+        List<Task> list = new ArrayList<Task>();
         Scanner in = new Scanner(System.in);
         while(!( line = in.nextLine()).equals("bye")) {
-                if(!line.equals("list")){
-                    list.add(line);
-                    System.out.println(separateLine + "\n" + "Added: " + line + "\n" + separateLine);
-                }else{
-                    System.out.println(separateLine);
-                    for(int i = 1; i <= list.size(); i++)
-                    {
-                        System.out.println(i + ". " + list.get(i-1));
-                    }
-                    System.out.println(separateLine);
+            if(line.equals("list")) {
+                System.out.println(separateLine+"\n"+"Here are the tasks in your list:");
+                //System.out.println(list.toString());
+                //Stream.of(list.toString()).forEach(System.out::println);
```

Comment:
Would be better for code quality if you could remove commented codes before a commit and push. 👍